test/integration/conftest.py

```diff
@@ -0,0 +1,58 @@
+import os
+import re
+import shutil
+import subprocess
+
+import git
+import pytest
+
+from hokusai.lib.common import ansi_escape
+
+
+TEST_GIT_REPO_NAME = 'hokusai-integration-test'
+
+
+def exit_pytest_if_not_minikube(context):
+  ''' exit pytest if context is likely not minikube '''
+  resp = subprocess.run(
+    f'kubectl --context {context} cluster-info',
+    capture_output=True,
+    shell=True,
+    text=True,
+    timeout=5
+  )
+  p = re.compile(r'Kubernetes control plane is running at https://(.*)')
+  m = p.search(ansi_escape(resp.stdout))
+  control_plane_address = m.groups()[0]
+  # 127.0.0.1 = local
+  # 192.168.x.y = circleci
+  if not (
+    control_plane_address.startswith('127.0.0.1') or
+    control_plane_address.startswith('192.168')
+  ):
+    msg = f'Error: {context} context possibly points to a real cluster. Is it a Minikube instance?'
+    pytest.exit(msg)
+
+def pytest_configure(config):
```

Comment:
This file performs setup for integration tests... perhaps a more descriptive file name than `conftest` would be helpful? Something like `test/integration/setup.py` for example? If you need pytest to auto-discover it you can tweak config for that perhaps.

Reply:
agreed. i am under the impression that the file has to be named `conftest.py`.

Reply:
Ah I see the [doc](https://docs.pytest.org/en/7.1.x/reference/fixtures.html#:~:text=The%20conftest.py%20file%20serves,pytest%20will%20automatically%20discover%20them) now, I haven't used one of those files before. We usually just load fixtures in from wherever we choose to place them. Ignore my comment in that case.